tutorials/geodetic_to_geocentric.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2019 The Boule Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# This code is part of the Fatiando a Terra project (https://www.fatiando.org)
+#
+r"""
+.. _geodetic_to_geocentric:
+
+Converting geodetic latitude and height into geocentric latitude and radius
+==============
+Both geocentric coordinate system and geodetic coordinate system are geographic 
+coordinate systems centred at the mass of the earth. In geocentric coordinate 
+system, the earth is modelled as a sphere. The geodetic coordinate system is 
+based on the reference ellipsoid considering the centrifugal force due to 
+the earth's rotation.
+
+The major difference between these two systems is the way how they define 
+latitude and height. In geocentric coordinate system, the geocentric latitude 
+is defined by the angle between the plane the equator and the line from earth 
+centre to a given point. The distance from the point to the earth centre is 
+geocentric radius :`R`. In geodetic coordinate system, the geodetic latitude is 
+the angle between the plane of the equator and the surface normal at a point 
+on the ellipsoid. The ellipsoid height :math:`h` is the height above the 
+ellipsoid surface, it also normal to that surface.
+
+.. image:: https://github.com/fatiando/boule/tree/master/doc/_static/coordinate.png
```

Comment:
```suggestion
.. figure:: ../_static/coordinate.png
    :alt: Depiction of geocentric and geodetic systems with associated coordinates.
    
    CAPTION SHOULD  BE ADDED HERE
```

Reply:
Using the `figure` directive is better because it allows for a caption. Would you mind adding the caption after merging this suggestion?